pkg/migrations/00004_update-gateway-envelopes.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE gateway_envelopes
```

Comment:
we also need a `down` SQL. Basically a revert.

Reply:
Introduced with the new sql files.